cloud_experts_tutorials/cloud-experts-rosa-hcp-activation-and-account-linking-tutorial.adoc

```diff
@@ -88,27 +112,34 @@ Both the Red{nbsp}Hat and AWS account numbers are shown on this screen.
 +
 If this is the first time you are using the {hybrid-console}, you will be asked to agree with the general managed services terms and conditions before being able to create the first ROSA cluster:
 +
+.Terms and conditions
++
 image::rosa-terms-conditions-9.png[]
 +
-Additional terms that need to be reviewed and accepted will be shown after clicking the *View Terms and Conditions* button:
+Additional terms that need to be reviewed and accepted are shown after clicking the *View Terms and Conditions* button:
++
+.Red{nbsp}Hat terms and conditions
 +
 image::rosa-terms-conditions-9-5.png[]
 +
 Submit your agreement once you have reviewed any additional terms when prompted at this time.
 
 . The {hybrid-console-second} provides a confirmation that AWS prerequisites were completed and lists the first steps needed for cluster deployment:
```

Comment:
. The {hybrid-console-second} provides a confirmation that AWS account setup was completed and lists the prerequisites for cluster deployment: